packages/react/src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import { Meta } from '@storybook/react';
+import Modal from './Modal';
+import Button from '../Button';
+
+const meta: Meta<typeof Modal> = {
+  title: 'Components/Modal',
+  component: Modal,
+};
+
+export default meta;
+
+const Template = props => {
+  const [open, setOpen] = useState(false);
```

Comment:
Let's change the default state to `true` (BTW same for the CSS project)

This is because the Visual Regression testing tool is not capturing a picture of the modal, but a picture of the button that opens the modal
So if there are any regressions on the Modal component, they won't be captured